lnst/RecipeCommon/Perf/Measurements/LinuxPerfMeasurement.py

```diff
@@ -7,81 +7,21 @@
 import re
 
 from lnst.Tests.LinuxPerf import LinuxPerf
-from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import (
-    BaseMeasurement,
+from lnst.RecipeCommon.Perf.Measurements.BaseMeasurement import BaseMeasurement
+from lnst.RecipeCommon.Perf.Measurements.Results import (
     BaseMeasurementResults,
+    LinuxPerfMeasurementResults,
```

Comment:
Wondering why we need the `BaseMeasurementResults`. I see that the measurement class uses this in type hinting, but wondering why we can't simply type-hint to `LinuxPerfMeasurementResults`.

Maybe a question for Jakub.

Reply:
@Kuba314 could you take a look?

Reply:
`LinuxPerfMeasurementResults` inherits from this. We can't remove the import.

We could change the type-hints to `LinuxPerfMeasurementResults` though. When I wrote this, I wanted to follow the same type-hints as are defined by the parent class, but that was illogical. We should change the type-hint on `collect_results` to `list[LinuxPerfMeasurementResult]` and *remove* the type-hint on the `results` variable in the same function since it's just noise and mypy figures this out on its own.

Reply:
@Kuba314  i think that's what the situation was before the moving refactor that @enhaut did here. right now this `LinuxPerfMeasurement` module seems to only use the `LinuxPerfMeasurementResults` imo.

Reply:
You're right. If the type gets changed, the import can of course be removed.